packages/admin/admin-icons/generate-icons.ts

```diff
@@ -6,48 +6,65 @@ import { pascalCase, pascalCaseTransformMerge } from "pascal-case";
 import * as path from "path";
 const eslint = new ESLint({ fix: true });
 
+type Icon = {
+    name: string;
+    path: string;
+    componentName: string;
+    deprecated: boolean;
```

Comment:
This could be optional. Then you don't have to set it for not deprecated icons.
```suggestion
    deprecated?: boolean;
```

Reply:
dd41ab5dd0eceefbaf23fec2b044cf9995bd1e68